scripts/influxd_wrapper.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+/usr/bin/influxd
+
+HOST=${INFLUXDB_HOSTNAME:-"localhost"}
+PORT=${INFLUXDB_HTTP_BIND_ADDRESS:-"8086"}
```

Comment:
This won't pick up any changes in `config.toml` that modify the port.

You could use `influxd print-config --key-name http-bind-address` to print the value that the server will use given the current environment. It will be a pseudo-full address, i.e. `:8086` as shorthand for `localhost:8086`

Reply:
Ah ok, I had wondered if `influxd` would populate those variables. the docs seemed a bit unclear. i'll fix

Reply:
So actually, in the context of this being run by systemd, I think any variables in the "environment file" will be present

Reply:
I would still recommend using `influxd print-config` here. It performs the same config & env resolution/merging as the top-level `influxd` command, so you won't need to worry about as many mismatches between what the script expects vs. what `influxd` actually does.